scripts/bitbucket.py

```diff
@@ -130,12 +139,18 @@ def post_result(url, user, password, verify, data, debug):
         err(json_pp(js))
 
     r = post_result(post_url, username, password, verify_ssl, js, debug)
-    if r.status_code != 204:
+    if driver == 'Bitbucket Server':
+        if r.status_code != 204:
+            sys.exit(1)
+    elif driver == 'Bitbucket Cloud':
+        if r.status_code != 200 and r.status_code != 201:
+            sys.exit(1)
+    else:
         sys.exit(1)
 
     status_js = {"version": {"ref": commit_sha}}
 
     if debug:
-        err("Returning to concourse:\n" + json_pp(status_js))
+        err("Returning to concourse:\n" + json.dumps(status_js))
```

Comment:
You mentioned `status_js` occasionally has a value of `None` and causes the line to fail. Could you write an `if` statement to check the value of `status_js` before passing it to `json.dumps()` so that the error doesn't occur?